src/core_plugins/input_control_vis/public/vis.less

```diff
@@ -0,0 +1,22 @@
+@import (reference) "~ui/styles/mixins.less";
+
+.inputControlVis {
+  width: 100%;
+
+  .inputRangeContainer {
+    padding: 15px 0;
+  }
+}
+
+visualization.input_control_vis {
+  overflow: visible;
+
+  .vis-container {
+    overflow: visible;
+  }
+}
+
+.Select-menu-outer {
```

Comment:
This is how you are making the pop ups appear in front of the other panels, right?  Just a thought but this is also why putting a generic select that wraps react-select in the uiFramework would be nice.  These kinds of css customizations can be kept in there, and then anywhere a drop down select needs to be used, it can take advantage of the custom css living in the uiFramework, not in the visualize plugin.  I'll leave it up to you though, can always be something tackled later, post k7.

btw, the capitalization is bc of react-select, not us right?